### ISSUE TITLE
[Bug] Some hostnames are cut off when using DASHDOT_SHOW_HOST=true

### ISSUE THREAD

**Issue by PlexSheep** [MauriceNino/dashdot#496]
### Description of the bug

The letter g (and probably some other letters) are cut off from the hostname, if `Dash.` is told to display it. This seems to occur with regardless of which browser I use, I tested this with Firefox and (ungoogled) Chromium on my Laptop as well as Firefox nightly on my Phone.

`Dash.` in this case is hosted on a public server, you can see it here: https://dashdot.ganggangoverflow.de/

Attatched below is an image of this very minor CSS Error:

![image](https://user-images.githubusercontent.com/58274330/209962803-b63cca12-71af-404e-a4f5-4294196b3dab.png)

### How to reproduce

- Have a Server with a hostname that contains a g like `ganggangoverflow`
- Host `Dash.` using docker-compose with the following `docker-compose.yml`
```yml
version: '3.5'

networks:
  default:
    name: REDACTED
    external: true

services:
  dash:
    image: mauricenino/dashdot:latest
    container_name: dashdot
    restart: unless-stopped
    environment:
      - DASHDOT_ALWAYS_SHOW_PERCENTAGES=true
      - DASHDOT_PAGE_TITLE=GangGangOverflow.de - Dash.
      - DASHDOT_SHOW_HOST=true
      - DASHDOT_NETWORK_LABEL_LIST=type,speed_up,speed_down,interface_speed,public_ip
      - DASHDOT_OVERRIDE_OS=Debian GNU/Linux 11 (bullseye)
      - DASHDOT_OVERRIDE_ARCH=x86_64
    privileged: true
    ports:
      - REDACTED:3001
    volumes:
      - /srv/dashdot:/mnt/host:
 ```

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.12.1
Dash: 4.5.7

Cwd: /app
Hash: afb9e0fcb3b1bad8ba684b751596d4a56c9e1dfc
In Docker: true
In Podman: false
In Docker (env): true
Done in 32.17s.
```

### What browsers are you seeing the problem on?

Firefox

### Where is your instance running?

Linux Server

### Additional context

Suggested Fix: Since the container box for "dash." and "hostname" is big enough, just move the font a tiny bit up.

**Comment by PlexSheep** [comment]
Sorry to bother you with a thing as minor as that.

**Comment by MauriceNino** [comment]
Hey there, thanks for reporting this! There is no such thing as a too minor bug ;)

I have fixed it locally, but I will need to sort out something with the release build, you will get a notification once this makes it into release!

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.6.0

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.

**Comment by PlexSheep** [comment]
Thanks :)
